oak/server/rust/oak_runtime/src/node/grpc/server.rs

```diff
@@ -133,16 +120,16 @@ impl GrpcServerNode {
         }?;
         self.channel_writer = Some(channel_writer);
 
-        warn!("Channel writer received: {:?}", self.channel_writer);
+        info!("Channel writer received: {:?}", self.channel_writer);
         Ok(())
     }
+}
 
-    /// Main Node worker thread.
-    fn worker_thread(mut self) {
-        let pretty_name = pretty_name_for_thread(&thread::current());
-
+/// Oak Node implementation for the gRPC server.
+impl Node for GrpcServerNode {
+    fn run(mut self: Box<Self>) {
```

Comment:
Not sure why `mut` is needed here?

Reply:
At this point the call to `self.init_channel_writer` below requires a `mut self`.  This gets removed in a later commit though.